0050-day-five/grooming/readme.md

```diff
@@ -0,0 +1,33 @@
+# Grooming
+
+## Content TODO's
+
+- Complete outline below
+- Provide guidance on criteria to determine if story can be estimated (see [definition of ready](../../team-agreements/definition-of-ready/readme.md))
+    - Examples of well defined acceptance criteria
+    - Can the story be tested as written
+    - Can it be completed within a sprint

```

Comment:
Not the right measuring stick. Instead, consider if a story can be naturally split, always. Only attempt to ensure that the *tasks* which comprise the story are < N story points.

From [Atlassian on Estimation](https://www.atlassian.com/agile/project-management/estimation):
> No individual task should be more than 16 hours of work. (If you're using story points, you may decide that, say, 20 points is the upper limit.) It’s simply too hard to estimate individual work items larger than that with a high degree of confidence. And that confidence is especially important for items at the top of the backlog. When something is estimated above your team's 16-hour (or 20-point) threshold, that's a signal to break it down into more granular pieces and re-estimate.

Reply:
again removed for now to let the general outline through. can add a recommendation here later once we come to consensus.